README.md

```diff
@@ -20,10 +20,12 @@ Translations
 
 Steps
 ------
-1. Run prepare.ps1 in an admin session of Powershell
+1. Ensure chocolatey is installed https://chocolatey.org/install
```

Comment:
If you're happy with your PR in this state i'll give it a test unless you want to chance some more things!

Reply:
Hi,

As I said, I learnt now how to make a "safe PR" using branches, so I will run more tests and submit again in the future on a new PR, ok